lti_consumer/migrations/0009_backfill-empty-string-config-id.py

```diff
@@ -27,7 +27,7 @@
 FROM
     lti_consumer_lticonfiguration
 WHERE
-    config_id = ""
+    config_id = '00000000-0000-0000-0000-000000000000'
```

Comment:
I think the behavior of this backfill would get altered by this, but this backfill only applies to MySQL in the first place (since PostgreSQL would be new, we'd never get into this weird state of needing to backfill in blank entries). So instead of altering the field we're searching on, maybe we could just skip the backfills entirely if we detect that the database being used is PostgreSQL?